src/api/getProductList.ts

```diff
@@ -0,0 +1,15 @@
+import { ProductListResponse, ProductParams } from '@/types/ProductItem';
```

Comment:
```suggestion
import type { ProductListResponse, ProductParams } from '@/types/ProductItem';
```

앞에 type 붙이면 더더더 좋을 것 같턴 넉낌쑤